examples/dbus-info/atomicClient.html

```diff
@@ -0,0 +1,220 @@
+<head>
+    <title>Atomic Client</title>
+    <meta charset="utf-8">
+    <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
+    <script src="../base1/jquery.js"></script>
+    <script src="../base1/cockpit.js"></script>
+</head>
+<body hidden>
+    <div id="ui" class="container-fluid" style='max-width: 400px'>
+        <table class="cockpit-form-table">
+            <tr>
+                <td><label class="control-label">Atomic</label></td>
+                <td><span id="current"></span></td>
+            </tr>
+            <tr>
+                <td><label class="control-label" for="new">Feature</label></td>
+                <td><input class="form-control" id="new" value=""></td>
+            </tr>
+            <tr>
+                <td><button class="btn btn-default btn-primary" id="Run">Run</button></td>
+		<td><span id="failure"></span></td>
+            </tr>
+        </table>
+    </div>
+
+    <div id="curtain" hidden>
+        Atomic D-Bus API not available.
+    </div>
+
+    <script>
```

Comment:
Lets put this in an external atomic-client.js script file. That way we can make the Content-Security-Policy strict.